### PR TITLE
(maint) Add command line options to ticketmatch

### PR DESCRIPTION
**Pull request by speedofdark**

### CODE REVIEW

File: ticketmatch.rb [hunk 1]
```diff
@@ -206,8 +243,13 @@ def inspect
...
 # collect the Jira information
 #
-jira_project_name          = ask('Enter JIRA project: ') {|q| q.default = 'PUP'}
-jira_project_fixed_version = ask('Enter JIRA fix version: ') {|q| q.default = "#{jira_project_name} #{git_to_rev}"}
+if jira_project_name == nil
+  jira_project_name = ask('Enter JIRA project: ') {|q| q.default = 'PUP'}
+end
+
+if jira_project_fixed_version == nil
+  jira_project_fixed_version = ask('Enter JIRA fix version: ') {|q| q.default = "#{jira_project_name} #{git_to_rev}"}
```
Comment (scottsolowild) [inline ticketmatch.rb]: This line is a bit long. I'd wrap by converting `{`/`}` to `do`/`end` with newlines for readability.

**Review by mcdonaldseanp — APPROVED** [review]
Other than Scott's readability comment, LGTM

**Review by smcclellan — CHANGES_REQUESTED** [review]
Can you squash all three of these commits into one? That way, in the commit history, there's no chance of the formatting fix or the README update being separated from the feature commit.

**Comment by speedofdark** [comment]
Updated.

**Review by smcclellan — APPROVED** [review]

**Comment by scottsolowild** [comment]
Just thought of this; might be good to add a `non-interactive` (or something like that) flag, so that if a pipeline misses a flag it won't stall indefinitely waiting for user input?

**Comment by speedofdark** [comment]
At the moment, it prompts for anything you omit on the command line. Will do some research and come back to you.

**Review by smcclellan — APPROVED** [review]

**Pull request merged**